src/main/java/com/yahoo/bullet/pubsub/rest/RESTSubscriber.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright 2018, Yahoo Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.pubsub.rest;
+
+import com.yahoo.bullet.pubsub.BufferingSubscriber;
+import com.yahoo.bullet.pubsub.PubSubException;
+import com.yahoo.bullet.pubsub.PubSubMessage;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.extern.slf4j.Slf4j;
+import org.asynchttpclient.AsyncHttpClient;
+import org.asynchttpclient.Response;
+
+@Slf4j
+public class RESTSubscriber extends BufferingSubscriber {
+    RESTPubSubConfig config;
```

Comment:
private for all these vars?

Reply:
I left them package so I can access them in tests.

Reply:
Expose them to tests by @Getter(AccessLevel.PACKAGE)?